config/locales/en.yml

```diff
@@ -1938,7 +1940,31 @@ en:
         tooltip: Metadata display settings
       spreadsheet_imports:
         dialog:
+          description: Importing a sample spreadsheet allows multiple samples with or without metadata fields to be added at once.
+          file: File
+          file_help: CSV, TSV, XLS or XLSX.
+          namespace:
+            group:
+              description_html: The spreadsheet is required to have columns for <b>Sample Name</b> and <b>Project PUID</b>, and optionally <b>Sample Description</b>.
+            project:
+              description_html: The spreadsheet is required to have a column for <b>Sample Name</b>, and optionally <b>Sample Description</b>.
+          project_puid_column: Project ID Column
+          sample_description_column: Sample Description Column (Optional)
+          sample_name_column: Sample Name Column
+          select_project_puid_column: Select a Project ID Column
+          select_sample_description_column: Select a Sample Description Column
+          select_sample_name_column: Select a Sample Name Column
           spinner_message: Importing samples, this might take a while...
+          submit_button: Import Samples
+          title: Upload Samples
+        errors:
+          description: 'The sample import completed with the following errors:'
+          ok_button: OK
+        success:
+          description: The sample spreadsheet has been processed.
+          ok_button: OK
+          problem_message: "%{path} %{message}"
+          problems: The following samples had problems while processing.
```

Comment:
Change this error message to `Errors were encountered while importing the following samples`

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/984/commits/1c30d7b0e5a3d125104197aab4e8b2066f557f74